CHANGES.md

```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.384 Jul 29 2024
```

Comment:
should be 0.0.385?

Reply:
good catch - it's actually 0.0.386 now since another MR got merged, but just fixed, thanks